Update tailwind.config.ts and slider.tsx

fengzhaohuaa opened the pull request:
主题相关配置

fengzhaohuaa committed:
Update tailwind.config.ts

fengzhaohuaa commented:
update主题相关配置到tailwind.config.ts

PLin2023 commented on tailwind.config.ts:
```diff
@@ -9,7 +9,69 @@ const config: Config = {
+        'system-ui',
+        '-apple-system',
+        'BlinkMacSystemFont',
+        '"Segoe UI"',
```
这个地方单引号中的双引号是必须的吗？

fengzhaohuaa replied:
yes yes ！

fengzhaohuaa replied:
https://tailwindcss.com/docs/font-family这里面有双引号滴

PLin2023 replied:
okok

fengzhaohuaa committed:
Create Footer.tsx

fengzhaohuaa committed:
Create Slider.tsx

fengzhaohuaa committed:
Add files via upload

fengzhaohuaa committed:
Update layout.tsx

fengzhaohuaa committed:
Update layout.tsx

PLin2023 commented on components/Slider.tsx:
Slider这个组件不行，数据不该写死在组件内部，而应该通过props传进来，包括你的当前索引，应该要能反馈到外部。参考我微信群里面的组件写法。

PLin2023 commented on app/layout.tsx:
layout的配色方案不对，参考https://nextui.org/docs/customization/dark-mode#using-next-themes，来实现。

fengzhaohuaa committed:
Update page.tsx

样式引入尝试页面

fengzhaohuaa committed:
Create index.css

tailwind.config.js中的颜色引用该文件内的颜色配置

fengzhaohuaa committed:
Update Slider.tsx

更改slider.tsx，将数据从外部传入如在pages.tsx中输入const images = ['./img/1.png', './img/2.png', './img/3.png'];，在return内部输入<Slider images={images} />即可实现图片轮播

fengzhaohuaa committed:
Update page.tsx

轮播图传入数据图片实验

PLin2023 reviewed (commented):
![image](https://github.com/PLin2023/cs-web/assets/86509543/5de45aef-5292-4c88-a6ee-65161bfa61f3)
如图，你的slider展现的信息会有如上这些，该slider的标题，显示的图片，跳转的地址。你需要加上类型限制如：
```ts
interface Slider {
    tittle?: string;
    image: string;
    url?: string;
}
```
另外你的Slider组件的props需要做类型约束

fengzhaohuaa commented:
嗷嗷嗷啊！！！sorry！me马上改！

fengzhaohuaa committed:
增加props的约束Slider.tsx

fengzhaohuaa commented:
'use client';
//引用轮播图的话加上这些代码
import React from 'react';
import Slider from '../components/Slider';
//轮播图信息
const images = ['./img/1.png', './img/2.png', './img/3.png'];
const title = 'Slider Title';
const image = './img/1.png';
const url = 'https://example.com';
export default function Home() {
  return (
    <>
      {/*轮播图引用*/}
      <Slider images={images} title={title} image={image} url={url} />
    </>
  );
}

fengzhaohuaa committed:
Update page.tsx

fengzhaohuaa committed:
更改slide的属性Slider.tsx

fengzhaohuaa committed:
引用轮播图page.tsx

fengzhaohuaa committed:
layout配色更改tailwind.config.ts

fengzhaohuaa commented:
<p className='text-text'>Make beautiful websites regardless of your design experience.</p>
          <p className='text-text dark'>Make beautiful websites regardless of your design experience.</p>
          测试引用文本颜色

fengzhaohuaa commented:
<p className='text-text'>Make beautiful websites regardless of your design experience.</p>
          <p className='text-text dark'>Make beautiful websites regardless of your design experience.</p>
          测试引用文本颜色
![image](https://github.com/PLin2023/cs-web/assets/97822086/4ec3c762-d241-449a-90cc-d6181d6ba817)

fengzhaohuaa committed:
Create ThemeSwitch.tsx日夜模式

暂定置于右侧悬浮按钮，以icon形式（可考虑后用文字的形式放导航栏）在pages.tsx中添加 <ThemeSwitcher />用于测试or在layout.tsx中用 <html lang='zh' className='light'>来测试

fengzhaohuaa committed:
Update tailwind.config.ts微调了一下

colors: {
        //fix颜色通常用于主题改变时颜色也不会发生改变，如底部信息栏的文字以及背景色
        fixwhite: '#fafafa',
        fixblue: '#291F64',
      },把这两个颜色放在colors里面

fengzhaohuaa committed:
Update Slider.tsx微调颜色

将有关颜色调整为主题配色方案

fengzhaohuaa committed:
Update Footer.tsx微调颜色

将文字和背景色调整为固定的主题配色方案（底蓝字白）

fengzhaohuaa committed:
Update page.tsx测试

用来测试主题色，轮播图，底部信息栏
